workshop/content/docs/advanced/administration.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Administration
+---
+
+# Administration
+
+## Overview
+
+pygeoapi provides an administration API (see the pygeoapi [documentation](https://docs.pygeoapi.io/en/latest/admin-api.html) for more information on how to enable, configure and use) in support of managing its configuration.  The API (not an OGC API) is implementated as a RESTful service to help create, update, replace or delete various elements of pygeoapi configuration.
+
+## User interface
+
+By design, pygeoapi does not provide a user interface to administer the configuration.  Given that the admin API exists, a few options can be considered for developing an admin UI:
```

Comment:
Should we say something about the existing UI for reading the configuration? https://github.com/geopython/pygeoapi/blob/master/pygeoapi/templates/admin/index.html

Reply:
Good catch!  I added a sentence to this effect.